assets/src/edit-story/components/library/panes/text/textSets/test/textSet.js

```diff
@@ -260,6 +260,9 @@ describe('TextSets', () => {
     insertTextSet.mockImplementation((elements) => elements);
     const { queryAllByRole } = setup(SET);
     const sets = queryAllByRole('listitem');
+
+    // Disable reason: it suggests to use toBeInDocument but here we really need the length instead.
+    // eslint-disable-next-line jest-dom/prefer-in-document
```

Comment:
@miina You'd actually want `queryByRole('listitem')` instead. It throws an error if there is more than one result.

Reply:
Actually, you'd want `getByRole('listitem')` because it throws an error if no elements match or if more than one match is found

Reply:
Good idea, thanks. Will add a comment for readability then that we're looking for 1 element only.